sources/packages/backend/apps/api/src/utilities/date-utils.ts

```diff
@@ -161,8 +162,22 @@ export function getDateOnlyFormat(date?: string | Date): string | undefined {
  * @param daysToAdd
  * @returns Date.
  */
-export const addDays = (date: Date | string, daysToAdd: number): Date => {
-  return dayjs(date).add(daysToAdd, "day").toDate();
+export const addDays = (daysToAdd: number, date?: Date | string): Date => {
+  return dayjs(date ? date : new Date())
+    .add(daysToAdd, "day")
+    .toDate();
+};
+
+/**
+ * @param date in which the search has to happen
+ * from 00:00 00:000 hrs to 23:59 59:999 hrs.
+ * @returns typeorm findOperator object between the date.
+ */
+export const dateEqualTo = (date?: Date): FindOperator<Date> => {
```

Comment:
IMO, maybe the function name can be something like `eligibleDate` or something else